src/mono/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.xml

```diff
@@ -177,4 +177,57 @@
       <attribute internal="RemoveAttributeInstances" />
     </type>
   </assembly>
+
+  <!-- Metadata attributes present in most core SDKs assemblies -->
+  <assembly fullname="*">
+    <type fullname="System.Reflection.AssemblyMetadataAttribute">
+      <attribute internal="RemoveAttributeInstances">
+        <argument type="System.Object">
+          <argument>Serviceable</argument>
+        </argument>
+      </attribute>
+    </type>
+    <type fullname="System.Reflection.AssemblyMetadataAttribute">
+      <attribute internal="RemoveAttributeInstances">
+        <argument type="System.Object">
+          <argument>PreferInbox</argument>
+        </argument>
+      </attribute>
+    </type>
+    <type fullname="System.Reflection.AssemblyMetadataAttribute">
+      <attribute internal="RemoveAttributeInstances">
+        <argument type="System.Object">
+          <argument>.NETFrameworkAssembly</argument>
+        </argument>
+      </attribute>
+    </type>
+    <type fullname="System.Reflection.AssemblyMetadataAttribute">
+      <attribute internal="RemoveAttributeInstances">
+        <argument type="System.Object">
+          <argument>RepositoryUrl</argument>
+        </argument>
+      </attribute>
+    </type>
+    <type fullname="System.Reflection.AssemblyMetadataAttribute">
+      <attribute internal="RemoveAttributeInstances">
+        <argument type="System.Object">
+          <argument>SourceCommitUrl</argument>
+        </argument>
+      </attribute>
+    </type>
+    <type fullname="System.Reflection.AssemblyMetadataAttribute">
+      <attribute internal="RemoveAttributeInstances">
+        <argument type="System.Object">
+          <argument>CommitHash</argument>
+        </argument>
+      </attribute>
+    </type>
+    <type fullname="System.Reflection.AssemblyMetadataAttribute">
+      <attribute internal="RemoveAttributeInstances">
+        <argument type="System.Object">
+          <argument>ReceiveHotReloadDeltaNotification</argument>
```

Comment:
From reading https://github.com/dotnet/sdk/pull/16401/files#r596499948, it sounds like this is just temporary and is going away. Maybe we should remove this entry?

cc @stephentoub @pranavkm